### PR TITLE
Added logic to view switch 

### LINKED ISSUE #5886: [ UI ] ViewSwitch acts weird
#### Current Behavior

Currently  we have a ```ViewSwitch``` to switch ```grid``` & ```table``` view for ```application/filter/design``` .
Suppose if I am on ```table``` view ,If I again press the table view button we get ```blank``` section. 

ll

#### Expected Behavior

We should correctly handle the state of  ```ViewSwitch```.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #5886

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/ViewSwitch.js [hunk 1]
```diff
@@ -8,7 +8,13 @@ function ViewSwitch({ view, changeView }) {
       size="small"
       value={view}
       exclusive
-      onChange={(_, newView) => changeView(newView)}
+      onChange={(_, newView) => {
+        if (newView !==null){
+          changeView(newView)
+        }
+
```
Comment (Chadha93) [inline ui/components/ViewSwitch.js:15]: formatting pleasae...

**Review by Chadha93 — COMMENTED** [review]
A video/demo in pr description will be always appreciated on a UI change.  Please add if you would :)

**Comment by Chadha93** [comment]
@harkiratsm would you please update the DCO? Let's merge after that. Thanks for the demo today. :1st_place_medal:

**Review by warunicorn19 — APPROVED** [review]
Looking good!!

**Pull request merged**